content/running_bookwyrm/install-prod-dockerless.md

```diff
@@ -76,16 +79,24 @@ GRANT ALL PRIVILEGES ON DATABASE bookwyrm TO bookwyrm;
 
 - Migrate the database schema by running `venv/bin/python3 manage.py migrate`
 - Initialize the database by running `venv/bin/python3 manage.py initdb`
+<<<<<<< HEAD
 - Compile the themes by running `venv/bin/python3 manage.py compile_themes`
 - Create the static files by running `venv/bin/python3 manage.py collectstatic --no-input`
 - If you wish to use an external storage for static assets and media files (such as an S3-compatible service), [follow the instructions](/external-storage.html) until it tells you to come back here
+=======
+- Create the static by running `venv/bin/python3 manage.py collectstatic --no-input`
+- If you wish to use an external storage provider for static assets and media files (such as an S3-compatible service), [follow the instructions](/external-storage.html) until it tells you to come back here
+>>>>>>> e2b5f577 (Update dockerless install docs with contrib's systemd services)
 - Create and setup your `bookwyrm` user
     - Make the system bookwyrm user:
         `useradd bookwyrm -r`
     - Change the owner of your install directory to bookwyrm:
         `chown -R bookwyrm:bookwyrm /opt/bookwyrm`
     - You should now run bookwyrm related commands as the bookwyrm user:
         `sudo -u bookwyrm echo I am the $(whoami) user`
+- Configure and start BookWyrm's `systemd` services:
+    - Copy the service configurations by running `cp contrib/systemd/*.service /etc/systemd/system/`
+    - Start the services with `systemctl start bookwyrm bookwyrm-worker bookwyrm-scheduler`
```

Comment:
Don't you need to enable them also so that they start on restart?

`sudo systemctl enable` ...

Reply:
You are right! Updated this